### ISSUE TITLE
Fabric 0.14.5 - Datapack couldn't be loaded

### ISSUE THREAD

**Issue by Nekkok** [MoonflowerTeam/pollen#41]
### What version are you seeing the problem on?

1.18.2

### Describe the issue

```markdown
I'm sorry to inform again, but last time I didn't clarfied that this error was happening on Fabric with Enhanced Mushrooms installed.
1. Install the mod.
2. Open game, loads fine.
3. Start new world > warn appears.

Thank you, I'm eager to play this with my modpack n_n
(log is cut because it was much for it lol)
```

### latest.log

```shell
[Render thread/WARN]: Failed to load datapacks, can't proceed with server load
java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot invoke "gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager$SidedReloader.getCompleteFuture()" because "gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager.serverReloader" is null
	at Not Enough Crashes deobfuscated stack trace.(1.18.2+build.3) ~[?:?]
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at net.minecraft.client.MinecraftClient.createSaveLoader(MinecraftClient:2124) ~[?:?]
	at net.minecraft.client.MinecraftClient.startIntegratedServer(MinecraftClient:1972) ~[?:?]
	at net.minecraft.client.MinecraftClient.createWorld(MinecraftClient:1941) ~[?:?]
	at net.minecraft.client.gui.screen.world.CreateWorldScreen.createLevel(CreateWorldScreen:286) ~[?:?]
	at net.minecraft.client.gui.screen.world.CreateWorldScreen.method_19922(CreateWorldScreen:230) ~[?:?]
	at net.minecraft.client.gui.widget.ButtonWidget.onPress(ButtonWidget:40) ~[?:?]
	at net.minecraft.client.gui.widget.PressableWidget.onClick(PressableWidget:16) ~[?:?]
	at net.minecraft.client.gui.widget.ClickableWidget.mouseClicked(ClickableWidget:121) ~[?:?]
	at net.minecraft.client.gui.ParentElement.mouseClicked(ParentElement:537) ~[?:?]
	at net.minecraft.client.Mouse.method_1611(Mouse:94) ~[?:?]
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen:492) ~[?:?]
	at net.minecraft.client.Mouse.onMouseButton(Mouse:94) ~[?:?]
	at net.minecraft.client.Mouse.method_22686(Mouse:165) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor:103) ~[?:?]
	at net.minecraft.client.Mouse.method_22684(Mouse:165) ~[?:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) ~[lwjgl-glfw-3.2.2.jar:?]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar:?]
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101) ~[lwjgl-glfw-3.2.2.jar:?]
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem:204) ~[client-intermediary.jar:?]
	at net.minecraft.client.util.Window.swapBuffers(Window:310) ~[?:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1147) ~[?:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:737) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:236) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:460) [fabric-loader-0.14.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.5.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager$SidedReloader.getCompleteFuture()" because "gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager.serverReloader" is null
	at gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager.getServerCompleteFuture(ResourceModifierManager.java:121) ~[pollen-fabric-1.4.1+1.18.2.jar:?]
	at net.minecraft.loot.LootManager.prepare(LootManager:1533) ~[?:?]
	at net.minecraft.resource.JsonDataLoader.prepare(JsonDataLoader:21) ~[?:?]
	at net.minecraft.resource.SinglePreparationResourceReloader.method_18791(SinglePreparationResourceReloader:11) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

**Comment by jaacksondev** [comment]
Duplicate of #34

**Comment by jacksonhardaway** [comment]
Fixed in dev, unreleased

**Comment by SpookLad** [comment]
Is there a solution for this issue yet, or do I have to wait for a new version to release?

**Comment by jacksonhardaway** [comment]
It's been fixed on GitHub and a new release will happen soon. Currently waiting on some upstream issues to be resolved.

**Comment by jacksonhardaway** [comment]
Fixed in Pollen 1.4.2
